gantry/models/build.py

```diff
@@ -0,0 +1,255 @@
+import json
+import logging
+import re
+from datetime import datetime
+
+import aiosqlite
+
+from gantry.util.gitlab import GitlabClient
+from gantry.util.misc import insert_dict, setattrs, spec_variants
+from gantry.util.prometheus import (
+    IncompleteData,
+    PrometheusClient,
+    process_resources,
+    process_usage,
+)
+
+
+class Build:
+    def __init__(
+        self,
+        status: str,
+        name: str,
+        id: int,
+        start: str,
+        end: str,
+        retries: int,
+        ref: str,
+    ):
+        self.status = status
+        self.name = name
+        self.id = id
+        self.start = datetime.fromisoformat(start).timestamp()
+        self.end = datetime.fromisoformat(end).timestamp()
+        self.retries = retries
```

Comment:
That should be `num_retries`, sorry for the vagueness. I thought it would be potentially useful to keep track of the number of times a job has been retried for future work.

Though, now I'm starting to question how this works. I need to figure out how GL tracks what counts as a retry...like what is the status of a job that is a retry of a retry? Either way, I think that's not important now. If you think num of retries wouldn't be necessary to track, happy to remove.

Reply:
Removing for now, can revisit in the future if we decide it's helpful for tracking OOMs.